webpack/assets/javascripts/bundle.js

```diff
@@ -40,7 +40,7 @@ window.tfm = Object.assign(window.tfm || {}, {
   hostgroups: require('./foreman_hostgroups'),
   hosts: require('./foreman_hosts'),
   httpProxies: require('./foreman_http_proxies'),
-  toastNotifications: require('./foreman_toast_notifications'),
+  toastNotifications: require('./react_app/common/foreman_toast_notifications'),
```

Comment:
Why? this is used outside of react.

Reply:
cc  @amirfefer

Reply:
?

Reply:
@amirfefer ?

Reply:
Both of us don't remember why I moved it so i'm moving it back. added foremanJS alias to access it from Katello